jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectionProvider.kt

```diff
@@ -306,6 +303,7 @@ class CawsConnectionProvider : GatewayConnectionProvider {
                                                             label(message("caws.workspace.connection.failed")).applyToComponent {
                                                                 font = JBFont.regular().asBold()
                                                             }
+                                                            label("Please ensure you are logged into the correct account.")
```

Comment:
formatting is weird
![image](https://github.com/aws/aws-toolkit-jetbrains/assets/742829/ff91c0de-e19b-4a6c-9370-c420dddf8509)

Reply:
Yes, this was supposed to be in brackets. Will fix this